portfolio/src/main/webapp/script.js

```diff
@@ -30,3 +30,21 @@ function addRandomFact() {
   factContainer.innerText = fact;
 }
 
+/** 
+    Sets tab buttons display content 
+*/
+function openTab(tab) {
+    let tabcontent;
+
+    // Hide each tab 
+    tabcontent = document.getElementsByClassName("tabcontent"); // get each tab labeled tab content
```

Comment:
The "Hide each tab" comment is good. It concisely explains the following block of code.

But I'd remove the "get each tab labeled tab content" comment, which doesn't add much.